app/views/taxons/confirm_delete.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>You are about to delete "<%= taxon.title %>"</h1>
+
+<%= render partial: 'deletion_warning',
+  locals: { taxon: taxon, children: children, tagged: tagged } %>
+
+<% if children.any? %>
```

Comment:
What do you think of adding a message in case there aren't any children to display?